marketplacetests/marketplace/app.py

```diff
@@ -66,7 +66,9 @@ def offline_message_text(self):
 
     def wait_for_notification_message(self, message):
         element = self.marionette.find_element(*self._notification_locator)
-        Wait(self.marionette).until(lambda m: element.text == message)
+        Wait(self.marionette).until(lambda m: element.text == message,
+                                    message='Notification not found. Expected: %s | Found: %s'
+                                            % (message, element.text))
```

Comment:
I don't think this will do what you expect. The message will be constructed when the call to `until` is first made, and so the values may not reflect the state at the point the timeout occurs. If you want to add debug for investigation it might be better to define a new function that includes log messages.